visualization/app/codeCharta/ui/metricChooser/metricChooser.po.ts

```diff
@@ -1,4 +1,19 @@
 export class MetricChooserPageObject {
+	public async openHeightMetricChooser() {
+		await expect(page).toClick("height-metric-chooser-component md-select", { timeout: 3000 })
+		await page.waitForSelector(".md-select-menu-container.ribbonBarDropdown.md-active")
+	}
+
+	public async clickOnHeightMetricSearch() {
+		await expect(page).toClick("md-backdrop", { button: "middle", timeout: 3000 })
+
+		await expect(page).toClick(".metric-search.height-metric", { timeout: 3000 })
+	}
+
+	public async isMetricChooserVisible() {
+		return page.waitForSelector(".md-select-menu-container.ribbonBarDropdown.md-active")
```

Comment:
```suggestion
		return page.waitForSelector(".md-select-menu-container.ribbonBarDropdown.md-active", { visible: true })
```

Otherwise this might return true, because it's still in the DOM, but invisible.